apis/beacon/blocks/blinded_blocks.yaml

```diff
@@ -32,6 +32,7 @@ post:
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Altair.SignedBeaconBlock"
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Bellatrix.SignedBlindedBeaconBlock"
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Capella.SignedBlindedBeaconBlock"
+            - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Deneb.SignedBlindedBeaconBlock"
```

Comment:
I think this is incorrect. We should publish blobs as well. So we need a `SignedBlindedBeaconBlockWithBlobs` type that wraps a `SignedBlindedBeaconBlock` and an array of `SignedBlobSidecar` (which is not defined in this PR), and use that type here.

Reply:
@rkapka you're right. This change is made in @realbigsean 's PR here https://github.com/ethereum/beacon-APIs/pull/302
Apologies for the messy diff, we're all kinda dependent on the [deneb rename branch](https://github.com/ethereum/beacon-APIs/pull/289) being merged, which should hopefully happen soon.

Reply:
This has been addressed in #302